src/Support/Utils/GlobalIdUtil.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support\Utils;
```

Comment:
Since this has to do with execution, how about we put it into src/Execution/Utils?

Reply:
@spawnia Sounds like a good idea to me, thanks for the feedback!!